scripts/gen-edpm-kustomize.sh

```diff
@@ -86,10 +86,16 @@ cat <<EOF >>kustomization.yaml
 EOF
 fi
 
+# NOTE(gibi): /spec/services/11 is nova today but it might
+# change in the future. So this is fragile. However kustomize
+# does not provide a way to replace a list item by value.
```

Comment:
maybe it is better to use sed instead of kustomize for this

Reply:
yeah I can try to add a bit of sed-ing to this script